kolibri/plugins/learn/assets/src/views/BookmarkPage.vue

```diff
@@ -170,6 +179,16 @@
       findFirstEl() {
         this.$refs.resourcePanel.focusFirstEl();
       },
+      stopMainScroll(sidePanelVisible) {
+        const bookmarkElement = this.$refs.bookmarkWrapper;
+        if (sidePanelVisible) {
+          bookmarkElement.style.position = 'fixed';
+          bookmarkElement.style.width = '84.65%';
```

Comment:
Why was the width setting necessary here? It wasn't needed in the previous PR.

Reply:
Will try this as well